src/javascript/app/pages/user/new_account/new_account_form_config/personal-details-config.js

```diff
@@ -85,20 +89,23 @@ const getPersonalDetailsConfig = ({ account_settings, residence_list }) => {
             section      : 'tax',
             supported_in : ['maltainvest'],
             default_value: false,
+            is_immutable : isImmutable('tax_identification_confirm'),
             rules        : ['req'],
         },
         {
             id           : 'account_opening_reason',
             section      : 'account_opening_reason',
             supported_in : ['iom', 'malta', 'maltainvest'],
             default_value: account_settings.account_opening_reason || '',
+            is_immutable : isImmutable('account_opening_reason'),
             rules        : ['req'],
         },
         {
             id           : 'accurate_answer_warning',
             section      : '',
             supported_in : ['maltainvest'],
             default_value: '',
+            is_immutable : isImmutable('accurate_answer_warning'),
             rules        : [],
         },
     ];
```

Comment:
SVG file change in this PR  is to fix circleci. CircleCi was failing since the below svg was not formatted with svgo.